Jokes_app/.gitignore

```diff
@@ -0,0 +1,3 @@
+#Ignoring node modules
+/node_modules
+package-lock.json
```

Comment:
Minor detail, but package-lock.json should be a committed file. This freezes the versions of your npm dependencies when other developers use your code and run `npm install`.

Reply:
Done.